src/manifests_workflow/input_manifests.py

```diff
@@ -95,76 +94,44 @@ def update(
 
             logging.info(f"Checking {self.name} {branches} branches")
             for branch in branches:
-                c = min_klass.checkout(
-                    path=os.path.join(work_dir.name, self.name.replace(" ", ""), branch),
+                min_component = min_klass.checkout(
+                    path=os.path.join(work_dir.name, self.prefix, branch),
                     branch=branch,
                 )
 
-                version = c.version
+                version = min_component.version
                 logging.info(f"{self.name}#{branch} is version {version}")
                 if version not in main_versions.keys():
-                    main_versions[version] = [c]
-
-            if component_klass is not None:
-                # components can increment their own version first without incrementing min
```

Comment:
Looks like this logic was not right and not useful that's why it was removed? since components won't be able to increment without min

Reply:
This was working but only limited to OpenSearch and OpenSearch-Dashboards, please correct me if I am wrong? @peterzhuamazon